manual/sphinx/developer_docs/code_layout.rst

```diff
@@ -345,8 +345,6 @@ The current source code files are:
 
 - sys
 
-   - :doc:`boutcomm.cxx<../_breathe_autogen/file/boutcomm_8cxx>`
```

Comment:
This is `boutcomm.cxx` rather than `comm_group.cxx`

Reply:
Sorry, there is a line referring to comm_group in there. I'll fix, thanks for spotting!